react-components/src/architecture/base/concreteCommands/PointCloudFilterCommand.ts

```diff
@@ -82,19 +85,39 @@ export class PointCloudFilterCommand extends BaseFilterCommand {
       pointCloud.setClassVisible(c.code, !isAllChecked);
     }
   }
+
+  // ==================================================
+  // INSTANCE METHODS
+  // ==================================================
+
+  private getPointCloud(): CognitePointCloudModel | undefined {
+    if (this._modelId === undefined || this._revisionId === undefined) {
+      return undefined;
+    }
+    for (const pointCloud of this.renderTarget.getPointClouds()) {
+      if (this._modelId === pointCloud.modelId && this._revisionId === pointCloud.revisionId) {
+        return pointCloud;
+      }
+    }
+    return undefined;
```

Comment:
This function could be rewritten as `return this.renderTarget.getPointClouds().find(pointCloud => pointCloud.modelId === this._modelId && pointCloud.revisionId === this._revisionId)`. It returns `undefined` if no applicable result is found